### PR TITLE
[common] Bump Flink version to 1.16.0

### PR DESCRIPTION
**Pull request by ruanhang1993**
Bump Flink version to 1.16.0

### CODE REVIEW

File: flink-cdc-e2e-tests/src/test/java/com/ververica/cdc/connectors/tests/utils/FlinkContainerTestEnvironment.java [hunk 1]
```diff
@@ -121,9 +121,11 @@ public abstract class FlinkContainerTestEnvironment extends TestLogger {
 
     @Parameterized.Parameters(name = "flinkVersion: {0}")
     public static List<String> getFlinkVersion() {
-        return Arrays.asList("1.13.6", "1.14.4", "1.15.2");
+        return Arrays.asList("1.13.6", "1.14.4", "1.15.2", "1.16.0");
     }
 
+    private static final List<String> FLINK_VERSION_SCALA_212 = Arrays.asList("1.15.2", "1.16.0");
```
Comment (leonardBang) [inline flink-cdc-e2e-tests/src/test/java/com/ververica/cdc/connectors/tests/utils/FlinkContainerTestEnvironment.java:127]: `FLINK_VERSION_WITH_SCALA_212`

File: flink-cdc-e2e-tests/pom.xml [hunk 2]
```diff
@@ -31,6 +31,7 @@ under the License.
         <flink-1.13>1.13.6</flink-1.13>
         <flink-1.14>1.14.4</flink-1.14>
         <flink-1.15>1.15.2</flink-1.15>
+        <flink-1.16>1.16.0</flink-1.16>
```
Comment (leonardBang) [inline flink-cdc-e2e-tests/pom.xml:34]: We need update our supported version in document as well

**Comment by leonardBang** [comment]
@ruanhang1993 please Run 'mvn spotless:apply' to fix these violations

File: docs/content/about.md [hunk 3]
```diff
@@ -31,7 +31,7 @@ The following table shows the version mapping between Flink<sup>®</sup> CDC Con
 | <font color="DarkCyan">2.0.*</font> | <font color="MediumVioletRed">1.13.*</font> |
 | <font color="DarkCyan">2.1.*</font> | <font color="MediumVioletRed">1.13.*</font> |
 | <font color="DarkCyan">2.2.*</font> | <font color="MediumVioletRed">1.13.\*</font>, <font color="MediumVioletRed">1.14.\*</font> |
-| <font color="DarkCyan">2.3.*</font> | <font color="MediumVioletRed">1.13.\*</font>, <font color="MediumVioletRed">1.14.\*</font>, <font color="MediumVioletRed">1.15.\*</font> |
+| <font color="DarkCyan">2.3.*</font> | <font color="MediumVioletRed">1.13.\*</font>, <font color="MediumVioletRed">1.14.\*</font>, <font color="MediumVioletRed">1.15.\*</font>, <font color="MediumVioletRed">1.16.0</font> |
```
Comment (leonardBang) [inline docs/content/about.md:34]: The guava we should use `30.1.1-jre-16.0` in pom.xml,  and could you create  issue for the failed tests?

**Review by leonardBang — CHANGES_REQUESTED** [review]
Thanks @

File: docs/content/about.md (hunk 3 above)
Comment (ruanhang1993) [reply docs/content/about.md:34]: Fixed. I will retry the tests later. It seems no problem locally.

**Review by leonardBang — APPROVED** [review]
Thanks @ruanhang1993 for the update, LGTM

**Pull request merged**